samples/plugins/sqlite_extract/manifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='sqlite_extract' superclass='odbc' plugin-version='0.0.0' name='SQLite Extract-Only Taco' version='18.1'>
+   <vendor-information>
+      <company name="Example"/>
```

Comment:
From the other samples:
      <company name="Company Name"/>
      <support-link url = "http://example.com"/>